Source/WebCore/platform/graphics/FontCascade.cpp

```diff
@@ -657,11 +657,6 @@ FontCascade::CodePath FontCascade::codePath(const TextRun& run, std::optional<un
 
     // FIXME: https://bugs.webkit.org/show_bug.cgi?id=150791: @font-face features should also cause this to be complex.
 
-#if !USE(FONT_VARIANT_VIA_FEATURES) && !USE(FREETYPE)
-    if (run.length() > 1 && (enableKerning() || requiresShaping()))
```

Comment:
We don't have support for kerning in our simple code path (`Font::applyTransforms` is only implemented for `CoreText`) -- I want to see the impact on layout tests of that change though.